modules/mono/glue/GodotSharp/GodotSharp/Core/Basis.cs

```diff
@@ -3,6 +3,21 @@
 
 namespace Godot
 {
+    /// <summary>
+    /// Specifies which order Euler angle rotations should be in.
+    /// When composing, the order is the same as the letters. When decomposing,
+    /// the order is reversed (ex: YXZ decomposes Z first, then X, and Y last).
+    /// </summary>
+    public enum EulerOrder
+    {
+        XYZ,
+        XZY,
+        YXZ,
+        YZX,
+        ZXY,
+        ZYX
+    };
```

Comment:
Is this used anywhere outside Basis? If not, we may want to move it inside the Basis struct, like `Vector3.Axis`.

Reply:
The same values are used in Node3D already, but I don't think it's typed as the enum (just int). Also, the plan is to use it in Quaternion soon, see https://github.com/godotengine/godot/pull/66747 and https://github.com/aaronfranke/godot/tree/quat-euler-order